src/App.js

```diff
@@ -6,24 +6,19 @@ import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
-const App = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
+import { PostList } from './conponents/postList';
```

Comment:
Better to fix the name of the folder.

```suggestion
import { PostList } from './components/postList';
```